components/Report/basic_usage.rst

```diff
@@ -0,0 +1,129 @@
+Basic Usage
+===========
+
+.. note::
+    First of all you need to register your data fetcher and renderer.
+    For more detailed information go to :ref:`Registry`_.
+
+.. _Registry: http://docs.sylius.org/en/latest/components/Registry/basic_usage.html
+
+.. _component_report_data-fetcher_delegating-data-fetcher:
+
+DelegatingDataFetcher
+---------------------
+
+This service allows you to delegate fetching to your custom registered service.
+Your data fetcher class should implement `DataFetcherInterface`_.
+
+.. _DataFetcherInterface: http://api.sylius.org/Sylius/Component/Report/DataFetcher/DataFetcherInterface.html
+
+.. code:: php
+
+    <?php
+
+    use Sylius\Component\Report\DataFetcher\DelegatingDataFetcher;
+    use Sylius\Component\Registry\ServiceRegistry;
```

Comment:
put that `use` on the top, and make all of them would sorted nicely, alphabetically?.